extensions/notification/NotificationQueuePopulator.js

```diff
@@ -125,8 +125,16 @@ class NotificationQueuePopulator extends QueuePopulatorExtension {
                 key,
                 eventType,
             });
+            const pushedToTopic = new Map();
             // validate and push kafka message foreach destination topic
             this.notificationConfig.destinations.forEach(destination => {
+                const internalTopic = destination.internalTopic ||
```

Comment:
nit: this variable is not always "internal" topic, but really just the topic : so may be renamed...